### ISSUE TITLE
Do not bundle styles in index.js and let us import the separated styles.css file

### ISSUE THREAD

**Issue by DavidRouyer** [euvl/vue-js-modal#587]
#### Problem: 

I'm using tailwind with vue-js-modal:
```vue
<template>
  <modal
    :name="this.name"
    :adaptive="true"
    :scrollable="true"
    width="600"
    height="auto"
    classes="bg-white rounded-lg shadow-xl transform transition-all sm:w-full p-4"
  >
  </modal>
</template>
```
It works fine in dev mode but my custom styles got overridden in production, because the styles are inlined in `dist/index.js` (see the capture below) therefore they take precedence over my custom classes.

#### Solution:
Strip CSS classes from `dist/index.js` and let us import the separated styles.css file wherever we want. So I can do:
```typescript
import "vue-js-modal/dist/styles.css";
import "@/assets/css/tailwind.css"; // My custom styles
```

#### Version:

2.0.0-rc.3

#### Example & screenshots:
![Capture](https://user-images.githubusercontent.com/1556750/86894459-61e9db00-c103-11ea-91e7-0e233cbe517c.PNG)

**Comment by euvl** [comment]
You can do that if you import nocss file. It is explained in in documentation.

**Comment by DavidRouyer** [comment]
The only nocss file I see is the `ssr.nocss` file which is (as the name seems to suggest) only useful for SSR scenarios as it throw an exception ` e._ssrNode is not a function` if I try to use it in a traditional browser scenario.

**Comment by brfrth** [comment]
@euvl This doesn't work. ssr.nocss does not work.

**Comment by euvl** [comment]
Thanks for your PR @DavidRouyer, i've published `2.0.0-rc.5` that should have your build

**Comment by euvl** [comment]
Will be closing this for now, ping me if you think this should stay open
